### ISSUE TITLE
Different behaviour with nix develop --command in file vs cli

### ISSUE THREAD

**Issue by locallycompact** [NorfairKing/feedback#12]
This works as expected:

```
feedback -- nix develop --command "cabal test all"
```

However,

```
loops:
  dev: nix develop --command "cabal test all"
```

followed by

```
feedback dev
```

results in

```
16:39:18 starting   nix develop --command "cabal test all"
/tmp/nix-shell.XMvQzr: line 1679: exec: cabal test all: not found
```

**Comment by NorfairKing** [comment]
I was able to reproduce this like this: 

```
loops:
  echo: nix-shell --command "echo 'hello world'"
```

**Comment by NorfairKing** [comment]
@locallycompact I thought I fixed the problem but I may have fixed a different problem. Could you check please?

**Comment by locallycompact** [comment]
Does not work for me.

**Comment by NorfairKing** [comment]
@locallycompact Seems to work for me. Could you send me an example without cabal? Maybe with some simple common command(s)?

**Comment by NorfairKing** [comment]
Double-checked at the airport, actually already fixed.
TL;DR, run this instead:

```
feedback -- nix develop --command cabal build all
```
